examples/FaxSend.java

```diff
@@ -0,0 +1,30 @@
+import com.dropbox.sign.ApiException;
+import com.dropbox.sign.Configuration;
+import com.dropbox.sign.api.*;
+import com.dropbox.sign.auth.*;
+import com.dropbox.sign.model.*;
+
+import java.util.List;
+
+public class Example {
+    public static void main(String[] args) {
+        var apiClient = Configuration.getDefaultApiClient()
+            .setApiKey("YOUR_API_KEY");
+
+        var faxApi = new FaxApi(apiClient);
+
+        var data = new FaxCreateRequest()
+            .areaCode(209)
+            .country("US");
```

Comment:
Add same values as from `examples/FaxSend.cs`

Files can be added with `.addFilesItem(new File("example_fax.pdf"))`